pkg/operator/ceph/file/controller.go

```diff
@@ -183,6 +183,7 @@ func (c *FilesystemController) ParentClusterChanged(cluster cephv1.ClusterSpec,
 
 func (c *FilesystemController) onDelete(obj interface{}) {
 	filesystem, err := getFilesystemObject(obj)
+
```

Comment:
remove the extra line? the common pattern is to check for err != nil on the line immediately after, so this looks odd